### ISSUE TITLE
Error: The bot is online but doesn't reply

### ISSUE THREAD

**Issue by hubertdungen** [Zero6992/chatGPT-discord-bot#202]
win10+python3.11.2+[chatGPT-discord-bot](https://github.com/Zero6992/chatGPT-discord-bot) latest

Following error while trying to run:

```
PS C:\Users\huber\source\repos\gpt-unleashed> py main.py
2023-02-25 17:41:58 INFO     discord.client logging in using static token
2023-02-25 17:41:59 INFO     discord.gateway Shard ID None has connected to Gateway (Session ID: fe92d89d6b304272441d95829c655a99).
2023-02-25 17:42:01 INFO     src.bot -> Send starting prompt with size 11
2023-02-25 17:42:01 ERROR    src.bot -> Error while sending starting prompt: Error communicating with OpenAI
Traceback (most recent call last):
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Lib\site-packages\aiohttp\connector.py", line 980, in _wrap_create_connection
    return await self._loop.create_connection(*args, **kwargs)  # type: ignore[return-value]  # noqa
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Lib\asyncio\base_events.py", line 1112, in create_connection
    transport, protocol = await self._create_connection_transport(
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Lib\asyncio\base_events.py", line 1145, in _create_connection_transport
    await waiter
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Lib\asyncio\sslproto.py", line 574, in _on_handshake_complete
    raise handshake_exc
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Lib\asyncio\sslproto.py", line 556, in _do_handshake
    self._sslobj.do_handshake()
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Lib\ssl.py", line 979, in do_handshake
    self._sslobj.do_handshake()
ssl.SSLCertVerificationError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:992)

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Lib\site-packages\openai\api_requestor.py", line 587, in arequest_raw
    result = await session.request(**request_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Lib\site-packages\aiohttp\client.py", line 536, in _request
    conn = await self._connector.connect(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Lib\site-packages\aiohttp\connector.py", line 540, in connect
    proto = await self._create_connection(req, traces, timeout)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Lib\site-packages\aiohttp\connector.py", line 901, in _create_connection
    _, proto = await self._create_direct_connection(req, traces, timeout)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Lib\site-packages\aiohttp\connector.py", line 1206, in _create_direct_connection
    raise last_exc
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Lib\site-packages\aiohttp\connector.py", line 1175, in _create_direct_connection
    transp, proto = await self._wrap_create_connection(
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Lib\site-packages\aiohttp\connector.py", line 982, in _wrap_create_connection
    raise ClientConnectorCertificateError(req.connection_key, exc) from exc
aiohttp.client_exceptions.ClientConnectorCertificateError: Cannot connect to host api.openai.com:443 ssl:True [SSLCertVerificationError: (1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:992)')]

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\huber\source\repos\gpt-unleashed\src\bot.py", line 108, in send_start_prompt
    responseMessage = await responses.handle_response(prompt)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huber\source\repos\gpt-unleashed\src\responses.py", line 11, in handle_response
    response = await chatbot.ask(message)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Lib\site-packages\revChatGPT\Official.py", line 232, in ask
    completion = await self._get_completion(
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Lib\site-packages\revChatGPT\Official.py", line 213, in _get_completion
    return await openai.Completion.acreate(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Lib\site-packages\openai\api_resources\completion.py", line 45, in acreate
    return await super().acreate(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Lib\site-packages\openai\api_resources\abstract\engine_api_resource.py", line 217, in acreate
    response, _, api_key = await requestor.arequest(
                           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Lib\site-packages\openai\api_requestor.py", line 300, in arequest
    result = await self.arequest_raw(
             ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Lib\site-packages\openai\api_requestor.py", line 604, in arequest_raw
    raise error.APIConnectionError("Error communicating with OpenAI") from e
openai.error.APIConnectionError: Error communicating with OpenAI
2023-02-25 17:42:02 INFO     src.bot -> GPT-Unleashed#3372 is now running!
```

**Comment by RocketGod-git** [comment]
Hi Hubert, good to see you here. 😉

**Comment by Zero6992** [comment]
duplicate of #178

The most likely situation is that OpenAI is blocked on your network, and you may be in a region where the use of ChatGPT is banned or using a VPN

**Comment by hubertdungen** [comment]
@RocketGod-git Hi! Nice to see you too xD
@Zero6992 nope I can use ChatGPT. I have used another GPT pack for discord but in JS / Node and it worked. I was trying your project but it immediately gave me an error when I ran it. I will try that solution.

**Comment by hubertdungen** [comment]
@Zero6992 

Ok just detected another error, maybe it's the reason. I tried to refresh everything, download a clean version of the repository. 
When I "pip install -r requirements.txt"

it retrieves this error:
```
PS C:\Users\huber\source\repos\gtp-unleashed_2> pip install -r requirements.txt
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Scripts\pip.exe\__main__.py", line 4, in <module>
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Lib\site-packages\pip\_internal\cli\main.py", line 9, in <module>
    from pip._internal.cli.autocompletion import autocomplete
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Lib\site-packages\pip\_internal\cli\autocompletion.py", line 10, in <module>
    from pip._internal.cli.main_parser import create_main_parser
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Lib\site-packages\pip\_internal\cli\main_parser.py", line 9, in <module>    
    from pip._internal.build_env import get_runnable_pip
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Lib\site-packages\pip\_internal\build_env.py", line 19, in <module>
    from pip._internal.cli.spinners import open_spinner
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Lib\site-packages\pip\_internal\cli\spinners.py", line 9, in <module>
    from pip._internal.utils.logging import get_indentation
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Lib\site-packages\pip\_internal\utils\logging.py", line 29, in <module>
    from pip._internal.utils.misc import ensure_dir
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Lib\site-packages\pip\_internal\utils\misc.py", line 42, in <module>
    from pip._internal.exceptions import CommandError, ExternallyManagedEnvironment
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Lib\site-packages\pip\_internal\exceptions.py", line 18, in <module>
    from pip._vendor.requests.models import Request, Response
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Lib\site-packages\pip\_vendor\requests\__init__.py", line 149, in <module>
    from . import packages, utils
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Lib\site-packages\pip\_vendor\requests\utils.py", line 24, in <module>
    from . import certs
  File "<frozen importlib._bootstrap>", line 1178, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1149, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Lib\site-packages\wrapt\importer.py", line 177, in _exec_module
    notify_module_loaded(module)
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Lib\site-packages\wrapt\wrappers.py", line 578, in __call__
    return self._self_wrapper(self.__wrapped__, self._self_instance,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Lib\site-packages\wrapt\decorators.py", line 470, in _synchronized
    return wrapped(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Lib\site-packages\wrapt\importer.py", line 136, in notify_module_loaded
    hook(module)
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Lib\site-packages\certifi_win32\wrapt_pip.py", line 35, in apply_patches
    import certifi
  File "<frozen importlib._bootstrap>", line 1178, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1149, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Lib\site-packages\wrapt\importer.py", line 177, in _exec_module
    notify_module_loaded(module)
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Lib\site-packages\wrapt\wrappers.py", line 578, in __call__
    return self._self_wrapper(self.__wrapped__, self._self_instance,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Lib\site-packages\wrapt\decorators.py", line 470, in _synchronized
    return wrapped(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Lib\site-packages\wrapt\importer.py", line 136, in notify_module_loaded
    hook(module)
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Lib\site-packages\certifi_win32\wrapt_certifi.py", line 20, in apply_patches
    certifi_win32.wincerts.CERTIFI_PEM = certifi.where()
                                         ^^^^^^^^^^^^^^^
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Lib\site-packages\certifi\core.py", line 37, in where
    _CACERT_PATH = str(_CACERT_CTX.__enter__())
                       ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Lib\contextlib.py", line 137, in __enter__
    return next(self.gen)
           ^^^^^^^^^^^^^^
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Lib\importlib\resources\_common.py", line 80, in _tempfile
    os.write(fd, reader())
                 ^^^^^^^^
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Lib\importlib\resources\abc.py", line 76, in read_bytes
    with self.open('rb') as strm:
         ^^^^^^^^^^^^^^^
  File "C:\Users\huber\AppData\Local\Programs\Python\Python311\Lib\importlib\resources\_adapters.py", line 141, in open
    raise FileNotFoundError("Can't open orphan path")
FileNotFoundError: Can't open orphan path
```
